pyugrid/__init__.py

```diff
@@ -9,7 +9,8 @@
 
 from .ugrid import UGrid
 from .uvar import UVar
+from . import grid_io
 
 __version__ = '0.1.7'
 
-__all__ = ['UGrid', 'UVar']
+__all__ = ['UGrid', 'UVar', 'grid_io']
```

Comment:
also -- does __all__ do anything useful if you don't do "import *" -- which you should never do :-). Not that it hurts, but why bother?